website/src/segment.tsx

```diff
@@ -116,9 +130,8 @@ export const AnnotatedForm = (
         </div>
         {p.segment.simplePhonetics ? (
```

Comment:
Make sure to change these render conditions since you aren't using `simplePhonetics` directly in rendering now.